meson.build

```diff
@@ -260,10 +260,12 @@ cc = meson.get_compiler('c')
 pkgconfig = import('pkgconfig')
 check_compilation_sh = find_program('tools/meson-check-compilation.sh')
 
-cxx = find_program('c++', required : false)
-if cxx.found()
-        #  Used only for tests
-        add_languages('cpp')
+if get_option('tests') != 'false'
```

Comment:
This isn't worth the complication. There really is very little cost for the failing (or succeeding) check.

Reply:
Currently, if cpp isn't found, meson will exit.  Some compilers might not have C++, hence the check.

Reply:
Please be precise: do you mean cpp or c++? I'm pretty sure cpp is required for cc to work, and c++ is optional.

Reply:
For cross-compiling environments, there is a possibility where cxx exists but cpp does not, in that instance, this check will fail.

Reply:
OK, let's add the check. I'm not sure if you're right, but I guess it's not that important.

But please fix the indentation to 8 spaces.

Reply:
Awesome! I will submit a new pull request in a few minutes.